frontend/new-admin-vm/Install-CAM.ps1

```diff
@@ -1557,7 +1559,57 @@ brokerLocale=en_US
 						if( !(($registerMachineResult.code -eq 201) -or ($registerMachineResult.data.reason.ToLower().Contains("exists")))) {
 							throw ("Registering Machine failed. Result was: " + (ConvertTo-Json $registerMachineResult))
 						}
+						$machineId = ""
+						# Get the machineId
+						if( ($registerMachineResult.code -eq 409) -and ($registerMachineResult.data.reason.ToLower().Contains("already exist")) ) {
+							# Deployment is already registered so the deplymentId needs to be retrieved
+							$registeredMachine = ""
+							try {
+								$registeredMachine = Invoke-RestMethod -Method Get -Uri ($camSaasBaseUri + "/api/v1/machines") -Body $machineRequest -Headers $tokenHeader
+								$machineId = $registeredMachine.data.machineId
+							} catch {
+								if ($_.ErrorDetails.Message) {
+									$registeredMachine = ConvertFrom-Json $_.ErrorDetails.Message
+									throw ("Getting Deployment ID failed. Result was: " + (ConvertTo-Json $registeredMachine))
+								} else {
+									throw $_
+								}								
+							}
+						} else {
+							$machineId = $registerMachineResult.data.machineId
+						}
 						Write-Host "Machine has been registered succesfully with Cloud Access Manager"
+						
+						# Register User Entitlement to Machine
+						# Get User Guid for Domain User
+						$DCSession = New-PSSession $using:dcvmfqdn -Credential $using:DomainAdminCreds
```

Comment:
localhost didn't work? I'm okay wither way just wanted to know if you tried it.

Reply:
I can try - I just copied what we were doing above

Reply:
No I was just wondering if you had tried. I want to eventually reduce the amount of remote powershelling, but we can cross that bridge later. Not a good experiment for the last day in a sprint.